api/apps/api/src/migrations/api/1685458802957-AddFeatureTagTable.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFeatureTagTable1685458802957 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE TABLE project_feature_tags (
+      id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+      project_id uuid NOT NULL references projects(id) ON DELETE CASCADE,
+      feature_id uuid NOT NULL references features(id) ON DELETE CASCADE,
+      tag varchar NOT NULL,
+      last_modified_at timestamp NOT NULL default now()
+    );`);
+
+    await queryRunner.query(`
+      CREATE UNIQUE INDEX unique_project_feature ON project_feature_tags (project_id, feature_id);
```

Comment:
Very minor, I suggest to use a different index name: we already have `unique_project_features on `features (feature_class_name, project_id) WHERE project_id IS NOT NULL`, and even though what each of these indexes does and where, I recommend not to use names that differ so slightly, as a favour to our future selves 😬 

what about this?
```suggestion
      CREATE UNIQUE INDEX single_tag_on_project_features ON project_feature_tags (project_id, feature_id);
```